justfile

```diff
@@ -77,8 +77,12 @@ elastic-stack-down:
   elastic-package stack down
 
 elastic-stack-connect-kind kind='kind-multi':
-  ID=$( docker ps --filter name={{kind}}-control-plane --format "{{{{.ID}}" ) && \
-  docker network connect elastic-package-stack_default $ID
+  #!/usr/bin/env sh
+
+  containers=$(docker ps | grep {{kind}} | awk '{ print $1 }')
+  for container in $containers; do
+    docker network connect elastic-package-stack_default $container
+  done
```

Comment:
I forgot to mention, we want to avoid adding scripts directly into our jusfile, but instead, calling them from our scripts dir (`./scripts/`) see more info in [here](https://github.com/elastic/security-team/issues/4291)